File: Fase 2/revistas-app/src/app/objetos/revista/Comentario.ts
export class Comentario {
    private _id: number;
    private _usuario: string;
    private _revista: number;
    private _numeroRevista: number;
    private _comentario: string;
    private _fechaPublicacion: Date;
    
    constructor(id: number, usuario: string, revista: number, numeroRevista: number, comentario: string, fechaPublicacion: Date) {
        this._id = id;
        this._usuario = usuario;
        this._revista = revista;
        this._numeroRevista = numeroRevista;
        this._comentario = comentario;
        this._fechaPublicacion = fechaPublicacion;
    }
    
    public get id(): number{
        return this._id
    }
    public get usuario(): string{
        return this._usuario
    }
    public get revista(): number{
        return this._revista
    }
    public get numeroRevista(): number{
        return this._numeroRevista
    }
    public get comentario(): string{
        return this._comentario
    }
    public get fechaPublicacion(): Date {
        return this._fechaPublicacion
    }
    
}

File: Fase 2/ServidorRevistas/src/main/java/objetos/Comentario.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package objetos;

import java.util.Date;

/**
 *
 * @author fernanrod
 */
public class Comentario {
    private Integer id;
    private String usuario;
    private Integer revista;
    private Integer numeroRevista;
    private String comentario;
    private Date fechaPublicacion;
    
    public Comentario(Integer id, String usuario, Integer revista, Integer numeroRevista, String comentario, Date fechaPublicacion) {
        this.id = id;
        this.usuario = usuario;
        this.revista = revista;
        this.numeroRevista = numeroRevista;
        this.comentario = comentario;
        this.fechaPublicacion = fechaPublicacion;
    }
    
    public Integer getId() {
        return id;
    }

    public String getUsuario() {
        return usuario;
    }

    public Integer getRevista() {
        return revista;
    }
    
    public Integer getNumeroRevista() {
        return numeroRevista;
    }

    public String getComentario() {
        return comentario;
    }
    public Date getFechaPublicacion() {
        return fechaPublicacion;
    }
}
